src/components/OptionsSelector/index.android.js

```diff
@@ -11,8 +10,6 @@ const OptionsSelector = forwardRef((props, ref) => (
     />
 ));
 
-OptionsSelector.propTypes = propTypes;
-OptionsSelector.defaultProps = defaultProps;
```

Comment:
Seemed unnecessary because we are passing these here -> https://github.com/Expensify/App/blob/01292ee0a9e53d0ee4724c541afb0e8629c17f96/src/components/OptionsSelector/BaseOptionsSelector.js#L22-L32